watertap/property_models/multicomp_aq_sol_prop_pack.py

```diff
@@ -330,14 +362,14 @@ def build(self):
             units=pyunits.m,
             doc="Stokes radius of solute",
         )
-        self.diffus_phase_comp = Param(
-            self.phase_list,
-            self.ion_set | self.solute_set,
+        self.molar_volume_comp = Param(
+            ["Liq"],
```

Comment:
Consider changing this to `phase_list`

Reply:
@lbibl I double checked and technically can be of any phase so it would be good for future usage, though the use in the H-L correlation for diffusivity is "molar volume at normal boiling point for solute." So then if changing the indexing to include `phase_comp`, would you think the naming convention across usage should also change to `molar_volume_phase_comp`? Note that the GAC model does call `properties_in[0].diffus_phase_comp["Liq", j]` so it should only call for the liquid input.

Reply:
Addressed in commit 2d72f93e